docs/insomnia/design-documents.md

```diff
@@ -5,40 +5,54 @@ category: "API Design"
 category-url: api-design
 ---
 
-Design Documents are a way to group and share API specifications, requests and unit tests within Insomnia. For working with collections, see Request Collections.
+Design Documents (also called Documents) are a way to create, group, and share API specifications, requests, and unit tests in Insomnia. 
 
-You can create, delete, and switch between Design Documents and Request Collections from the Dashboard, accessed from the upper left-hand corner of the application.
+Switch between your Documents and Collections via the Dashboard. Inside a Document or Collection, click on **Dashboard**. 
 
-## Create a new Design Document
+{:.alert .alert-primary}
+**Note**: If you don't see a Document or Collection you're looking for, ensure you're in the right [Project](/insomnia/projects) by clicking on the Insomnia dropdown menu.  
```

Comment:
Note this dropdown menu will only say "Insomnia" when the Insomnia/Default project is selected

![image](https://user-images.githubusercontent.com/4312346/133342337-fedeac1f-17e5-4783-a039-5afb3385568f.png)